release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1305,6 +1305,36 @@ This script removes unauthenticated subjects from the following cluster role bin
 // TODO: This known issue should carry forward to 4.9 and beyond!
 * The `oc annotate` command does not work for LDAP group names that contain an equal sign (`=`), because the command uses the equal sign as a delimiter between the annotation name and value. As a workaround, use `oc patch` or `oc edit` to add the annotation. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1917280[*BZ#1917280*])
 
+* If the installation program is unable to get all of the projects that are associated with the Google Cloud Platform (GCP) service account, the installation fails with a `context deadline exceeded` error message.
++
+This behavior occurs when the following conditions are met:
+
+** The service account has access to an excessive number of projects.
+** The installation program is run with one of the following commands:
+*** `openshift-install create install-config`.
```

Comment:
```suggestion
*** `openshift-install create install-config`
```
None of the sub-bullets are complete sentences, so they don't need to end with periods.